setup.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+
+# If a binary distribution exists for your OS and Python, congratulations.  You should
+# be able to simply install swig.
+#
+#
+# If you are doing a build from sources, you must also have have swig and a copy of
```

Comment:
"have have"

Reply:
I changed this whole thing, telling them to check out the README.developer isntead